notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -479,38 +617,6 @@
     ")"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": 11,
-   "id": "5f05808a-8a4a-4573-830e-456e13e1d787",
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "076706c6c356493695ffe524e36f3c80",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "VBox(children=(Dropdown(description='Device:', options=('CPU', 'GPU', 'AUTO'), value='CPU'), Checkbox(value=Tr…"
-      ]
-     },
-     "execution_count": 11,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "int8_compressed_weights = widgets.Checkbox(\n",
-    "    value=True,\n",
-    "    description='Use compressed weights',\n",
-    "    disabled=False\n",
-    ")\n",
-    "\n",
-    "widgets.VBox([device, int8_compressed_weights])"
```

Comment:
Add `device` row. After removing this block, there no device output widget.

Reply:
Thanks, fixed